app/views/unattended/report_templates/host_-_applied_errata.erb

```diff
@@ -46,11 +54,12 @@ require:
   version: 3.12.0
 -%>
 <%- report_headers 'date', 'hostname', 'erratum_id', 'erratum_type', 'status' -%>
-<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'), 
+<%- load_errata_applications(filter_errata_type: input('Filter Errata Type'),
                              include_last_reboot: input('Include Last Reboot'),
                              since: input('Since'),
                              up_to: input('Up to'),
-                             status: input('Status')
+                             status: input('Status'),
+                             host_filter: input('Host filter').presence
```

Comment:
`Host filter` vs `Hosts filter` causes `ERF01-2862 [Foreman::Renderer::Errors::UndefinedInput]: Rendering failed, no input with name "Host filter" for input macro found` when I try to render the report